lib/task-data/tasks/install-coreos.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     implementsTask: 'Task.Base.Os.Install',
     options: {
         // NOTE: user/pass aren't used by the coreos installer at the moment,
-        // but they are required values
+        // but they are required values for Task.Base.Os.Install
```

Comment:
The reason for the comment was that I found the pass-through of values in the declarative JSON blobs to be fairly confusion - not sure what's defined where, what's required, and what's not. The CoreOS installer, in fact, doesn't have any capacity to specify this - the most you can tweak there is the SSH key that allows you to log into the base account for their system. An outlier in the OS install space, but worth noting.